server/src/main/java/org/apache/druid/server/coordinator/CompactionSupervisorConfig.java

```diff
@@ -33,10 +34,12 @@
  */
 public class CompactionSupervisorConfig
 {
-  private static final CompactionSupervisorConfig DEFAULT = new CompactionSupervisorConfig(null);
+  private static final CompactionSupervisorConfig DEFAULT = new CompactionSupervisorConfig(null, null);
 
   @JsonProperty
   private final boolean enabled;
+  @JsonProperty
+  private final CompactionEngine defaultEngine;
```

Comment:
Nit: Maybe we should call this property just `engine`.

Reply:
Updated to `engine`.